views/trials-details.html

```diff
@@ -73,6 +73,19 @@
       </span>
       {{ add_discrepancy_icon_if_needed(trial, 'status') }}
     </li>
+    {% set primary_records = trial.records | selectattr("is_primary") | selectattr("last_verification_date") %}
```

Comment:
IMO this should belong to the trial presenter and have an associated test. See how the decorators are built in there (e.g. https://github.com/opentrials/opentrials/blob/master/presenters/trial.js#L61)